### PR TITLE
grpc section

### PR DESCRIPTION
**Pull request by drizuid**
add grpc support for the app

### CODE REVIEW

**Comment by drizuid** [comment]
thanks to thedinz

File: pydio-cells.subdomain.conf.sample [hunk 1]
```diff
@@ -41,6 +41,9 @@ server {
         set $upstream_port 8080;
         set $upstream_proto https;
         proxy_pass $upstream_proto://$upstream_app:$upstream_port;
+        if ($http_content_type = "application/grpc") {
+            grpc_pass grpcs://pydio-cells:33060;
```
Comment (Roxedus) [inline pydio-cells.subdomain.conf.sample:45]: Should be the $upstream_app:$upstream_port pattern.

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Comment by drizuid** [comment]
putting this on hold as I don't use pydio-cells and the [documentation ](https://pydio.com/en/docs/kb/deployment/running-cells-behind-nginx-reverse-proxy)implies that the port used is the normal 8080 port. need to get confirmation/testing from someone that uses this.
image

File: pydio-cells.subdomain.conf.sample (hunk 1 above)
Comment (aptalca) [reply pydio-cells.subdomain.conf.sample]: done

**Comment by aptalca** [comment]
upstream now defaults to 8080 for grpc, var is removed from our pydio-cells image in https://github.com/linuxserver/docker-pydio-cells/pull/17

**Review by aptalca — APPROVED** [review]

**Review by Roxedus — APPROVED** [review]

**Pull request merged**